cpu/kinetis_common/doc.txt

```diff
@@ -14,7 +14,7 @@
  *                  #define ADC_NUMOF           (1U)
  *                  #define ADC_0_EN            1
  *                  #define ADC_MAX_CHANNELS    1
- *                  
+ *
  *                  // ADC 0 configuration
```

Comment:
The problem is that the comment is inside a comment so it has to be either
// or / (space) *
On Mar 3, 2015 6:56 PM, "Peter Kietzmann" <notifications@github.com> wrote:

> In cpu/kinetis_common/doc.txt
> <https://github.com/RIOT-OS/RIOT/pull/2481#discussion_r25709523>:
>
> > @@ -14,7 +14,7 @@
> >   *                  #define ADC_NUMOF           (1U)
> >   *                  #define ADC_0_EN            1
> >   *                  #define ADC_MAX_CHANNELS    1
> > - *
> > + *
> >   *                  // ADC 0 configuration
>
> ... and following
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/2481/files#r25709523>.
>